Formula/neovim.rb

```diff
@@ -43,13 +43,13 @@ class Neovim < Formula
   end
 
   resource "lua-compat-5.3" do
-    url "https://github.com/keplerproject/lua-compat-5.3/archive/v0.7.tar.gz"
-    sha256 "bec3a23114a3d9b3218038309657f0f506ad10dfbc03bb54e91da7e5ffdba0a2"
+    url "https://github.com/keplerproject/lua-compat-5.3/archive/v0.9.tar.gz"
+    sha256 "ad05540d2d96a48725bb79a1def35cf6652a4e2ec26376e2617c8ce2baa6f416"
   end
 
   resource "luv" do
-    url "https://github.com/luvit/luv/releases/download/1.30.0-0/luv-1.30.0-0.tar.gz"
-    sha256 "5cc75a012bfa9a5a1543d0167952676474f31c2d7fd8d450b56d8929dbebb5ef"
+    url "https://github.com/luvit/luv/releases/download/1.34.1-1/luv-1.34.1-1.tar.gz"
```

Comment:
just curious, any reason to choose `1.34.1-1` instead of `1.36.0-0`?  https://github.com/luvit/luv/releases

Reply:
> While here, also updates some build dependencies, luv only up to 1.34.1-1 since later releases won't build. Would need to be investigated further and build process probably adjusted.